sonic_psu/psu_base.py

```diff
@@ -47,3 +47,75 @@ def get_psu_presence(self, index):
         """
         return False
 
+    def get_model(self, idx):
+        """
+        Retrieves the model number/name of a power supply unit (PSU) defined
+                by 1-based index <idx>
+        :param idx: An integer, 1-based index of the PSU of which to query model number
+        :return: String, denoting model number/name
+        """
+        return ""
+
+    def get_mfr_id(self, idx):
```

Comment:
can you give an example string for this id? Not sure the difference between mfr_id and serial number.

Reply:
mfr_id is the 'Manufacturer's ID'. Either name or some abbreviation which denotes the manufacturer. e.g. 
1. On Delta platforms, PSU mfr_id is 'DELTA'
2. On some Accton platforms, PSU mfr_id is '3Y POWER'
3. On soem Quanta platforms, PSU mfr_id is 'ARTESYN'